ledger/internal/cow.go

```diff
@@ -245,6 +245,10 @@ func (cb *roundCowState) BlockHdr(r basics.Round) (bookkeeping.BlockHeader, erro
 	return cb.lookupParent.BlockHdr(r)
 }
 
+func (cb *roundCowState) StateProofVerificationContext(stateProofLastAttestedRound basics.Round) (*ledgercore.StateProofVerificationContext, error) {
+	return cb.lookupParent.StateProofVerificationContext(stateProofLastAttestedRound)
```

Comment:
It could make sense to replace the recursion with `while` loop, however, more than 10 other methods of `roundCowState` behave in a similar manner (recursing through `lookupParent` until reaching the `roundCowBase`) so I don't see a particular reason to change the behavior just for this method.  

Also, since this behavior is very common for `roundCowState` there's no real need to document specifically how it works.